pom.xml

```diff
@@ -72,7 +72,11 @@
             <artifactId>commons-lang3</artifactId>
             <version>3.1</version>
         </dependency>
-
+        <dependency>
+            <groupId>com.github.dkpro</groupId>
+            <artifactId>dkpro-jwpl</artifactId>
+            <version>de.tudarmstadt.ukp.wikipedia-1.1.0</version>
+        </dependency>
```

Comment:
Is there a reason why this is done differently here than in the [upstream fork](https://github.com/diegoceccarelli/json-wikipedia/blob/master/pom.xml#L61-L85)?

Reply:
Sorry, I probably didn't explain myself very well above!

I'm not a fan of this syntax:
```
<version>de.tudarmstadt.ukp.wikipedia-1.1.0</version>
```
because `de.tudarmstadt.ukp.wikipedia-1.1.0` isn't a version. `1.1.0` is a version, `de.tudarmstadt.ukp.wikipedia` is an artifact ID. (I realise that's the fault of repo, but I still think it's wrong.)

Also, by depending on `dkpro-jwl` in this way, I suspect you're also pulling in stuff that you don't need: `dkpro-jwl` [includes `mwdumper`](https://github.com/dkpro/dkpro-jwpl/blob/master/pom.xml#L189-L193), for example.

I'd prefer the individual dependencies that we need from `de.tudarmstadt.ukp.wikipedia` from Maven Central in the same way as the [upstream repository](https://github.com/diegoceccarelli/json-wikipedia/blob/master/pom.xml#L61-L85) has done, but with version 1.1.